common/app/views/fragments/collections/popular.scala.html

```diff
@@ -10,7 +10,7 @@
 @import model.Pillar.RichPillar
 
 @defining(popular.size > 1){ isTabbed =>
-    <div id="popular-trails" class="fc-slice fc-slice--popular" data-link-name="most popular Test" data-test-id="popular-in">
+    <div id="popular-trails" class="fc-slice fc-slice--popular" data-link-name="most popular" data-test-id="popular-in">
```

Comment:
This was introduced here https://github.com/guardian/frontend/commit/55369d9e227ba96148ef2d4218f5a26b5d6a430e#diff-b0c567c53b12e6ff5427340b22ab52d69e8a8077e66c0e830016fa68ecf35020R10 in 2015. I think we can rename it.